devito/ir/support/vector.py

```diff
@@ -1,6 +1,6 @@
 from collections import OrderedDict
 
-from sympy import Basic
+from sympy import Basic, true
```

Comment:
i know it has been merged already. but just for learning: what's the importance of this? wouldn't it be considered an overkill?

Reply:
sympy booleans are "more powerful" than python booleans when it comes to comparing sympy expressions. Look for where it's used and try replace it with ``True`` , you'll see things start breaking (why?)

Reply:
i see. 
i would guess that's because something like 
'2 + 2' == '4'
could be evaluated as 'sympy.true'
but would not be considered 'True'?